### DISCUSSION TITLE
晚睡強迫症

### DISCUSSION THREAD

**Discussion by CPW0919** [1101-GS4719-C/Python-Community-GS4719-C#1319] [General]
白天不屬於自己的時間，晚上要加倍討回來。
可能是對睡覺有著莫名的罪惡感，認為睡覺就是浪費時間。這些人在白天的工作效率不佳，到了夜晚在罪惡感的催逼之下，試圖藉由熬夜對白天進行某種程度的補償；又或是經歷了白天的勞累，到夜晚靠著誘發式的亢奮活動，欲去除心理上的疲憊，這些人多沉溺在網路上，觀看網路視頻、追劇、聊天，或透過強勁節拍的音樂來抒發壓力，近年有個名詞指稱上述的現象為「報復性熬夜」。

以下為我認為不錯的解決方法:
1.創造假的熬夜環境: 晚睡強迫症的人睡前一定有一個固訂行程，像是追劇 聽歌，可以把做這些事的時間整個提前，並將室內燈光調暗騙過你的大腦
2.睡前寫好隔天代辦事項，準備好隔天的衣服: 蠻多人都會在床上翻來覆去思考隔天要做的事<只要提千把該做的是用白紙黑字寫好，就會產生一種了卻心事的安心感。
3.關上手機: 一定很有用 但我做不到 :(

總結，雖然我列了那麼多但我沒有成功過，祝我成功。

**Comment by harrycoconut** [comment]
這個跟本就是在講我吧！常常會覺得白天睡太晚而晚上一定要玩回來的報復性思維，謝謝你分享此篇報導，看了這篇報導後，我恍然大悟！這種報復性的熬夜持續的在摧殘我的人生，我今天起會慢慢改正我的睡覺習慣😌

**Comment by Jamesmoneya** [comment]
我試過第三點我覺得這是唯一最有用的方法，常常到了半夜已經很累了，但是睡覺之前還是會習慣性地滑一下，在心理學層面就是被制約了，尤其如果你已經看了一整晚的書，如果中途和最後都沒有划手機，那麼即使你身心已經非常的累了，你還是沒辦法睡著，你得腦袋那時候已經不是被你控制，而是手機，因此趕快戒斷手機吧，如果不行，至少在睡前不要用
，

**Comment by llh26** [comment]
我自己也有這種報復性心態，太早睡反而覺得怪怪的，常常躺在床上，久久無法入睡，最後還是拿起手機⋯，這種狀態真的好難受。不知道大家是否曾經有過這樣的經驗：讀書讀很晚，怕明天沒精神只好去睡覺，結果整個大失眠，整晚淺眠到比睡前還累，腦中只有「早知道就起來繼續讀」的想法，真的很怕再遇到這種情況，希望自己的睡眠品質可以有所提升。

**Comment by Miinteaa** [comment]
同意晚睡強迫症的說法。且晚睡真的會形成惡性循環😣因為晚睡又早起造成上課精神不佳，會不小心睡著，但睡飽了之後晚上又睡不著了。我能找到的解決方法就是按照自己的睡眠週期（1.5小時一個週期）計算自己的睡眠時間以外，強迫在某個時間點爬上床休息，希望我能夠順利找回正常的作息🥺

**Comment by ALAN30359704** [comment]
我覺得手機真的影響現代人太大了!以前我曾經聽過一個說法，在睡前看手機，會刺激大腦無法讓人進入休息的狀態。

**Comment by shuxian12** [comment]
我自己試過如果我在九點半到十點這段時間讀書，就會超級容易睡著，有時候就算看影片也會，所以現在都會在那個時間去洗澡，才不會讓自己想睡覺，但另一方面也是我自己覺得我早上的效率非常的低，所以我寧願晚點睡。也不想早點起床

**Comment by juliaouo** [comment]
好中肯...我習慣晚睡已經到覺得凌晨3點還很早的程度，天天4、5點睡是常態，即使隔天早8也是一樣，導致睡眠時間只有2、3小時，不過有時候真的是太怕事情做不完，在晚上解決比白天爬起來更有安心感，可能是因為離 dead line 遠一點XD
